### PR TITLE
Use a space-separated in timesyncd.conf

### PR DESCRIPTION
**Pull request by hfm**
According to timesyncd.conf(5) in man, NTP and FallbackNTP need a space-separated list: https://www.freedesktop.org/software/systemd/man/timesyncd.conf.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,8 +163,8 @@ It is possible to configure the default ntp servers in /etc/systemd/timesyncd.co
 ```puppet
 class{'::systemd':
   $manage_timesyncd => true,
-  $ntp_server          => '0.pool.ntp.org,1.pool.ntp.org',
-  $fallback_ntp_server => '2.pool.ntp.org,3.pool.ntp.org',
+  $ntp_server          => '0.pool.ntp.org 1.pool.ntp.org',
+  $fallback_ntp_server => '2.pool.ntp.org 3.pool.ntp.org',
```
Comment (raphink) [inline README.md]: How about passing an array instead?

Comment (hfm) [reply README.md]: $ntp_server and $fallback_ntp_server in systemd class now don't allow Array: https://github.com/camptocamp/puppet-systemd/blob/1.1.0/manifests/init.pp#L43-L44

Comment (hfm) [reply README.md]: IMO, it's better that systemd class allow $ntp_server and $ntp_fallback_server to use both Array and String. If you're okay, I'll do.

Comment (raphink) [reply README.md]: Oh ok. That's weird, cause the `timesyncd` class accepts arrays. @bastelfreak you originally wrote that class, what do you think?

Comment (ekohl) [reply README.md]: Sounds like an oversight and ```init.pp``` should match the types of ```timesyncd.pp```

Comment (bastelfreak) [reply README.md]: spaces are correct, not sure how I missed it. @hfm can you also allow array as datatype in the init.pp?

Comment (hfm) [reply README.md]: okay i'll change it.

Comment (hfm) [reply README.md]: I fixed it by 2c38f1a

**Pull request merged**